### PR TITLE
Table:   column resize accessibility issues 

### PR DESCRIPTION
**Pull request by rjavier-trimbler**
## Description
the column resize cursor is too tight to be accessible for users with dexterity problems and we include some padding for it 

References #1973 

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules
- [x] I have checked my code and corrected any misspellings

### CODE REVIEW

**Comment by apaddock** [comment]
@Roliver-Javier this PR does not accomplish the full 8px with 4px on each side for grabbing the column resize. Let's mark it as a draft PR since it's not quite there yet. 

@cjwinsor may have some suggestions on what else we can try. 

Right now it seems that we can only add additional pixels of touch target/hover target for resizing columns to the LEFT side of the column separate and not any to the right.

**Comment by cjwinsor** [comment]
@rjavier-trimbler @apaddock I was on a call with @dnarey yesterday and this came up and I showed him briefly the following as we reviewed how I had previously done it in e-builder's planning module.  It seemed to do the trick.

 - give the parent (td) `position: relative; overflow: visible;` 
 - give the handle `position: absolute; top: 0; right: -4px; width; 7px; z-index:1; height: 100%;`
 - give the div with the text in the header `overflow: hidden; text-overflow: ellipsis;`
 
We just weren't sure at the time if there was any accessibility concerns.  The current `focus-visible` styling is only a right border and should probably be an outline.  The reason its 7px wide is I was trying to visually center it when the outline was shown relative to the border.

**Comment by JTrimbleR** [comment]
Validated, working as expected.

image

image

File: stencil-workspace/src/components/modus-table/modus-table.scss [hunk 1]
```diff
@@ -156,25 +156,32 @@ table {
             }
           }
         }
-
+
```
Comment (cjwinsor) [inline stencil-workspace/src/components/modus-table/modus-table.scss:159]: run `npm run lint` to correct formatting issues

**Review by cjwinsor — CHANGES_REQUESTED** [review]

File: stencil-workspace/src/components/modus-table/modus-table.scss (hunk 1 above)
Comment (rjavier-trimbler) [reply stencil-workspace/src/components/modus-table/modus-table.scss]: done!

**Review by cjwinsor — APPROVED** [review]

**Pull request merged**